source/_integrations/opower.markdown

```diff
@@ -43,7 +43,7 @@ More than 175 utilities use Opower. Currently only the following utilities are s
   - PECO Energy Company (PECO)
   - Potomac Electric Power Company (Pepco)
 - Glendale Water and Power (GWP)
-- Mercury NZ Limited
+- ~Mercury NZ Limited~ (no longer supported, will be removed in 2025.9.0)
```

Comment:
To prevent people going to ask questions, can you change no longer supported into transitioned away from oPower?

Reply:
Done. I added the same text I have in the virtual integration page linking to the Mercury representative post so that users can take their complains there.